Library/Formula/mesos.rb

```diff
@@ -12,19 +12,22 @@ class Mesos < Formula
   depends_on :java => "1.7+"
   depends_on :macos => :mountain_lion
   depends_on "maven" => :build
-  # Use our Zookeeper for Yosemite and not the one shipped with Mesos
-  # Remove with next release.
-  # See https://github.com/Homebrew/homebrew/issues/32965
-  depends_on "zookeeper" if MacOS.version == :yosemite
+  depends_on :apr => :build
+  depends_on "subversion"
+  
+
+  needs :cxx11
 
   def install
     args = ["--prefix=#{prefix}",
             "--disable-debug",
             "--disable-dependency-tracking",
             "--disable-silent-rules",
+            "--with-svn=/usr/local/Cellar/subversion/1.8.11",
```

Comment:
Thanks!  committed. And thanks for the clear explanations!!

I get a sneaking suspicion that we will need to mess with apr.  When installing manually I get 
"-------------------------------------------------------------------
libapr-1 is required for mesos to build.
-------------------------------------------------------------------"

Would this mean altering the apr homebrew formula to install an 'include/' directory? or digging into the xcode library packages?